canopeum_backend/canopeum_backend/models.py

```diff
@@ -130,6 +136,39 @@ class Coordinate(models.Model):
     dd_longitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True)
     address = models.TextField(blank=True, null=True)
 
+    @staticmethod
+    def from_dms_lat_long(dms_latitude: str, dms_longitude: str):
+        dms_latitude_split = re.split(r"°|\'|\.|\"", dms_latitude)
```

Comment:
You could compile this regex a single time ahead of time using
```py
LAT_LONG_SEP = re.compile(r"°|\'|\.|\"")
```
(outside the class scope, at the top of the module)